packages/angular/ssr/package.json

```diff
@@ -29,7 +29,6 @@
     "@angular/platform-server": "19.0.0-next.1",
     "@angular/router": "19.0.0-next.1",
     "@bazel/runfiles": "^5.8.1",
```

Comment:
Question: Would `@bazel/runfiles` make more sense in the root `package.json`? I'm kinda surprised it's not there already.

Reply:
It is there: https://github.com/angular/angular-cli/pull/28228/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R86